src/pages/index.js

```diff
@@ -136,28 +125,110 @@ const editFormValidator = new FormValidator(
 );
 const addCardFormValidator = new FormValidator(validationSettings, addCardForm);
 
-editFormValidator.enableValidation();
+const editAvatarValidator = new FormValidator(
+  validationSettings,
+  avatarEditForm
+);
 
+editFormValidator.enableValidation();
 addCardFormValidator.enableValidation();
+editAvatarValidator.enableValidation();
+
+/* -------------------------------------------------------------------------- */
+/*                      Project 8 GENERATE INITIAL CARDS                      */
+/* -------------------------------------------------------------------------- */
+
+// Preview image Card
+const previewimagePopup = new PopupWithImage("#preview-image-modal");
 
 /* -------------------------------------------------------------------------- */
-/*                                  Project 8                                 */
+/*                                  Project 9                                 */
 /* -------------------------------------------------------------------------- */
 
-// Edit Profile Info
-const userInfo = new UserInfo({
+const api = new Api({
+  url: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "b32399ae-a567-415e-9d15-bc2048a1a730",
+    "Content-Type": "application/json",
+  },
+  response: (res) => {
+    if (res.ok) {
+      return res.json();
+    }
+    return Promise.reject(`Error: ${res.status}`);
+  },
+});
+
+// 1. Loading user information from the server
+api
+  .getUserInfo()
+  .then((user) => {
+    userInfo.setUserInfo({ name: user.name, about: user.about });
+    userID = user._id;
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+// 2. Loading cards from the server
+
+api
+  .getInitialCards()
+  .then((initialCards) => {
```

Comment:
You can receive all the initial data together using `Promise.all`, so you will save user's id and provide it to Card with constructor arguments and will be able to define if user is card's owner

```js
let userId;

Promise.all([api.getUserInfo(), api.getInitialCards()]).then(([user, cards]) => {
  userInfo.setProfileInfo(userData.name, userData.about);
  userId = user._id;  // set user id
  // initialize cards section
}).catch(() => console.log(...))
```